### PR TITLE
New OSDP version 1.1.0

### PR DESCRIPTION
**Pull request by proux01**
The CI will fail with all but Debian based systems because of the absence of depext for conf-csdp, c.f., previous version: https://github.com/ocaml/opam-repository/pull/13179

### CODE REVIEW

**Comment by proux01** [comment]
Pinging @ploc

**Comment by mseri** [comment]
The url does not work:
```
[ERROR] The sources of the following couldn't be obtained, aborting:
          - osdp.1.1.0:
            https://cavale.enseeiht.fr/osdp/osdp-1.1.0.tgz (curl: code 404 while downloading https://cavale.enseeiht.fr/osdp/osdp-1.1.0.tgz)
```

**Comment by ploc** [comment]
Can you use [ploc:osdp.1.1.0](https://github.com/ploc/opam-repository/tree/osdp.1.1.0)  ? I updated it to target the github repo. 

Or shall I create a fresh PR? Sorry, I am kinda new to this.

File: packages/conf-csdp/conf-csdp.1/opam [hunk 1]
```diff
@@ -1,11 +1,12 @@
 opam-version: "2.0"
 maintainer: "Pierre Roux"
 authors: "CSDP"
+license: "CPL"
 homepage: "https://github.com/coin-or/Csdp/wiki"
 build: ["sh" "-exc" "command -v csdp"]
 depexts: [
   ["coinor-csdp"] {os-family = "debian"}
-  ["csdp"] {os-distribution = "centos"}
+  ["csdp-tools"] {os-distribution = "centos" | os-distribution = "fedora" | os-distribution = "ol"}
```
Comment (mseri) [inline packages/conf-csdp/conf-csdp.1/opam:9]:
```suggestion
  ["csdp-tools"] {os-distribution = "centos" | os-distribution = "fedora" | os-distribution = "ol"}
  ["coin-or-csdp"] {os-distribution = "archlinux"}
```

**Comment by proux01** [comment]
@mseri thanks for your work (and your patience with my stupid mistakes here)

**Comment by mseri** [comment]
Thanks @dinosaure

**Pull request merged**